pricelist_cache/migrations/14.0.1.4.1/README.rst

```diff
@@ -0,0 +1,4 @@
+As it causes problems in other module's tests when creating pricelist items,
+and since the `base_automation` module won't be accessible in odoo community edition
+anymore (since 17.0), we drop the base automation record in favor of a hook in
```

Comment:
Not sure to understand that point. (module not accessible since V17).
see : https://github.com/odoo/odoo/tree/17.0/addons/base_automation